src/leaderboard/leaderboard.js

```diff
@@ -0,0 +1,82 @@
+import './leaderboard.css';
+
+import Button from '../components/Button/Button';
+import '../components/Button/Button.css';
+
+// funkcja renderująca stronę
+
+export function renderLeaderboard() {
+  podium();
+  sortScores();
+  localScores();
```

Comment:
To improve readability create one function (getScoreFromLocalStorage). It should retrieve highScore (instead line 54) and return it sorted to the main function.
Then you should pass it to localScores function (but it should be named to the convention - (showScore)